Problem: upgrade to serde_with 2 fails

```
error[E0433]: failed to resolve: could not find `display_fromstr` in `rust`
791
  --> common/src/node/cosmos_sdk.rs:54:20
792
   |
793
54 |     #[serde(with = "serde_with::rust::display_fromstr")]
794
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `display_fromstr` in `rust`
795
796
error[E0433]: failed to resolve: could not find `display_fromstr` in `rust`
797
  --> common/src/node/cosmos_sdk.rs:57:20
798
   |
799
57 |     #[serde(with = "serde_with::rust::display_fromstr")]
800
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `display_fromstr` in `rust`
801
802
For more information about this error, try `rustc --explain E0433`.
803
error: could not compile `defi-wallet-core-common` due to 2 previous errors
```

--- a/common/Cargo.toml
+++ b/common/Cargo.toml
@@ -36,7 +36,7 @@ rustc-hex = { version = "2", optional = true }
 secrecy = "0.8"
 serde = "1"
 serde_json = "1"
-serde_with = "1"
+serde_with = "2"
 siwe = { version = "0.4", optional = true }
 tendermint = "0.23"
 tendermint-proto = "0.23"

--- a/common/src/node/cosmos_sdk.rs
+++ b/common/src/node/cosmos_sdk.rs
@@ -7,6 +7,7 @@ use cosmos_sdk_proto::cosmos::{
 #[cfg(not(target_arch = "wasm32"))]
 use itertools::Itertools;
 use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, DisplayFromStr};
 use tendermint_rpc::{
     endpoint::broadcast::{tx_async, tx_commit, tx_sync},
     request, response,
@@ -49,6 +50,7 @@ pub enum RawRpcAccountResponse {
 }
 
 /// the raw account status data from the account API
+#[serde_as]
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct RawRpcAccountStatus {
     /// the protobuf type
@@ -59,10 +61,10 @@ pub struct RawRpcAccountStatus {
     /// the associated public key
     pub pub_key: Option<RawRpcPubKey>,
     /// the global account number
-    #[serde(with = "serde_with::rust::display_fromstr")]
+    #[serde_as(as = "DisplayFromStr")]
     pub account_number: u64,
     /// the sequence number / nonce
-    #[serde(with = "serde_with::rust::display_fromstr")]
+    #[serde_as(as = "DisplayFromStr")]
     pub sequence: u64,
 }
 

